lib/ui/formatPicker/index.ts

```diff
@@ -73,12 +73,11 @@ class FormatPicker extends BaseFloat {
     };
 
     const handleKeydown = (event) => {
-      const { key, shiftKey, metaKey } = event;
-
+      const { key, shiftKey, metaKey,ctrlKey } = event;
```

Comment:
One white space is needed before `ctrlKey`.

Reply:
> One white space is needed before `ctrlKey`.

yeap, I will fix it